cloudinstall/placement/ui/__init__.py

```diff
@@ -165,9 +165,6 @@ def build_widgets(self):
         self.deploy_ok_msg = ("\u2713 All the required OpenStack services are "
                               "placed on a machine, and you can now deploy.")
 
-        # Save a new placements configuration once all required services are
-        # placed.
-        self.placement_controller.save()
```

Comment:
Crap it didn't take my change there, I actually commented that out because we weren't passing in the yaml file object. This actually didn't exist in the prior implementation of the placements file so I'm going to remove this